### DISCUSSION TITLE
Can't get intall guides for open-simulation-interface for windows

### DISCUSSION THREAD

**Discussion by jingxinmingyue** [OpenSimulationInterface/open-simulation-interface#653] [Q&A]
In the linkage of https://github.com/OpenSimulationInterface/open-simulation-interface, in the bottom, there is saying "For Windows installation see [here](https://opensimulationinterface.github.io/osi-documentation/open-simulation-interface/doc/windows.html) for more information.", but the link seems not right and can't access.

**Comment by raue** [comment]
Yes, it's a pity, having a link at a prominent spot, which doesn't work. But digging into the documentation unveils some support for linux and windows: take look [here](https://opensimulationinterface.github.io/osi-documentation/#_setting_up_osi)
